packages/nuxt/src/pages/runtime/plugins/router.ts

```diff
@@ -142,6 +142,7 @@ const plugin: Plugin<{ router: Router }> = defineNuxtPlugin({
 
     const initialLayout = nuxtApp.payload.state._layout
     router.beforeEach(async (to, from) => {
+      nuxtApp.callHook('page:loading:start')
```

Comment:
Hooks always return a promise and not catching it can cause unhandled issues ++ losing ability of tracking async contexts which will be more important in the future. 

Is there an incompatibility that we are not awaiting on them?

Reply:
Yes, awaiting these seemed to hang the app - see https://github.com/nuxt/nuxt/actions/runs/7208947233/job/19638983604?pr=24010.

We should await if possible.

Reply:
I would personally await on it

Reply:
Shall we use `callHookParallel` instead? (https://github.com/unjs/hookable/blob/dc85dadf7624d171a186fe0ec662efeb5886d815/src/hookable.ts#L197C13-L203)